daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -75,19 +75,133 @@ Code | Description
 This API does not return any content.
 
+## Raise Event request
+
+For workflow components that support subscribing to external events, such as the Dapr Workflow engine, you can use the following "raise event" API to deliver a named event to a specific workflow instance.
+
+```http
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceID>/raiseEvent/<eventName>
+```
+
+{{% alert title="Note" color="primary" %}}
+ The exact mechanism for subscribing to an event depends on the workflow component that you're using. Dapr Workflow has one way of subscribing to external events but other workflow components might have different ways.
+
+{{% /alert %}}
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Use `dapr` for Dapr Workflows
+`instanceId` | Unique value created for each run of a specific workflow
+`eventName` | The name of the event to raise
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or underlying component
+
+### Response content
+
+None.
+
+## Pause workflow request
+
+Pause a running workflow instance.
+
+```http
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceId>/pause
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Use `dapr` for Dapr Workflows
+`instanceId` | Unique value created for each run of a specific workflow
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Error in Dapr code or underlying component
+
+### Response content
+
+None.
+
+## Resume workflow request
+
+Resume a paused workflow instance.
+
+```http
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceId>/resume
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Use `dapr` for Dapr Workflows
+`instanceId` | Unique value created for each run of a specific workflow
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Error in Dapr code or underlying component
+
+### Response content
+
+None.
+
+## Purge workflow request
+
+Purge the workflow state from your state store with the workflow's instance ID.
+
+```http
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceId>/purge
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Use `dapr` for Dapr Workflows
+`instanceId` | Unique value created for each run of a specific workflow
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Error in Dapr code or underlying component
+
+### Response content
+
+None.
+
 ### Get workflow request
```

Comment:
Heading at the wrong level.
```suggestion
## Get workflow request
```

Can we also add the status into the table that can be returned. Notably `Running`, `Terminated` etc
```suggestion
### Get workflow request
```